ffw/UIRPC.js

```diff
@@ -202,16 +202,17 @@ FFW.UI = FFW.RPCObserver.create(
                 'secondaryGraphic' in request.params ||
                 'softButtons' in request.params ||
                 'customPresets' in request.params) {
+
+                if (this.errorResponsePull[request.id].type === 'STATIC') {
+                  this.sendError(SDL.SDLModel.data.resultCode['UNSUPPORTED_RESOURCE'], 
+                                request.id, 
+                                request.method, 
+                                'Image of STATIC type is not supported on HMI. Request was not displayed');
```

Comment:
```suggestion
                                'Image of STATIC type is not supported on HMI. Request was not processed');
```

Reply:
@AKalinich-Luxoft reworked in https://github.com/LuxoftSDL/sdl_hmi/pull/2/commits/e27904fa6b64e62aa103b63dfb65fd7cef936b97